packages/p2p/src/components/order-details/payment-method-accordion-content.jsx

```diff
@@ -5,9 +5,12 @@ import './payment-method-accordion-content.scss';
 const PaymentMethodAccordionContent = ({ payment_method }) => {
     return (
         <React.Fragment>
-            {Object.entries(payment_method?.fields).map((field, key) => {
+            {Object.entries(payment_method?.fields).map(field => {
                 return (
-                    <div key={key} className='payment-method-accordion-content--field'>
+                    <div
+                        key={`${field[1].display_name}${field[1].value}`}
```

Comment:
Just the value here should do.

```suggestion
                        key={field[1].value}
```